src/test/versionbits_tests.cpp

```diff
@@ -151,6 +191,11 @@ class VersionBitsTester
     VersionBitsTester& TestActive() { return TestState(ThresholdState::ACTIVE); }
     VersionBitsTester& TestFailed() { return TestState(ThresholdState::FAILED); }
 
+    // non-delayed should be active; delayed should still be locked in
```

Comment:
nit: this comment seems to apply to `TestActiveLockedIn`, not `TestLockedInDelayed`.